{"message":"token error","time":1695048810607,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695051016634,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412668060,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412674736,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412674742,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412696706,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695412696710,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695412729599,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695412738072,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695412796648,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695413071754,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695413119871,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695413554355,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"column Users->UsersInConversations.id does not exist","time":1695413620570,"stackTrace":{"error":"SequelizeDatabaseError: column Users->UsersInConversations.id does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:953:12)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:21)"}}
{"message":"token error","time":1695416419153,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695416419172,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695416424443,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695416424453,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"Wrong password","time":1695416437149,"code":404,"stackTrace":{"error":"UserNotFoundError: Wrong password\n    at _callee4$ (/server/src/controllers/queries/userQueries.js:37:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)"}}
{"message":"Wrong password","time":1695416442127,"code":404,"stackTrace":{"error":"UserNotFoundError: Wrong password\n    at _callee4$ (/server/src/controllers/queries/userQueries.js:37:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)"}}
{"message":"Wrong password","time":1695416445441,"code":404,"stackTrace":{"error":"UserNotFoundError: Wrong password\n    at _callee4$ (/server/src/controllers/queries/userQueries.js:37:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)"}}
{"message":"token error","time":1695416446746,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695416446751,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"this email were already exist","time":1695416472213,"code":409,"stackTrace":{"error":"NotUniqueEmail: this email were already exist\n    at _callee2$ (/server/src/controllers/userController.js:52:12)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/userController.js:9:103)\n    at _throw (/server/src/controllers/userController.js:11:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695734349701,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695734371006,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695734371013,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"Wrong password","time":1695734378152,"code":404,"stackTrace":{"error":"UserNotFoundError: Wrong password\n    at _callee4$ (/server/src/controllers/queries/userQueries.js:37:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)"}}
{"message":"need token","time":1695735120460,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"user with this data didn`t exist","time":1695735127323,"code":404,"stackTrace":{"error":"UserNotFoundError: user with this data didn`t exist\n    at _callee2$ (/server/src/controllers/queries/userQueries.js:19:11)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/queries/userQueries.js:15:103)\n    at _next (/server/src/controllers/queries/userQueries.js:17:194)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"need token","time":1695735128330,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695735128336,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695737475238,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737475280,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737475291,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737477830,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737477835,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737477841,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737480894,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737480899,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737480904,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695737480910,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"need token","time":1695737670412,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695737672270,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695737672275,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695741874040,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695741874056,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695741879504,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695741879508,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695744309599,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695744310607,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695744310612,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695744310617,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695744310632,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.throw (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _throw (/server/src/middlewares/checkToken.js:5:291)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"need token","time":1695744329019,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695744329024,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695744330676,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1695744330681,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695749263452,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695749263473,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695749267468,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695749267472,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695749267475,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695749267485,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695824447657,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695824447692,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695824449565,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695824449568,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695824449571,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695824449577,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695830747058,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee2$ (/server/src/middlewares/checkToken.js:38:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:29:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/server/node_modules/express/lib/router/index.js:317:13)\n    at /server/node_modules/express/lib/router/index.js:284:7\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)\n    at next (/server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/server/node_modules/express/lib/router/index.js:174:3)"}}
{"message":"token error","time":1695830747077,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee2$ (/server/src/middlewares/checkToken.js:38:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:29:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/server/node_modules/express/lib/router/index.js:317:13)\n    at /server/node_modules/express/lib/router/index.js:284:7\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)\n    at next (/server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/server/node_modules/express/lib/router/index.js:174:3)"}}
{"message":"token error","time":1695830854105,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695830854143,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695830858095,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695830858099,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695830858102,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695830858110,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1695830873932,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"token error","time":1695835864400,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695835864450,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695835866864,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695835866868,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695835866877,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695835866884,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"column \"conversationsId\" does not exist","time":1695836607189,"stackTrace":{"error":"SequelizeDatabaseError: column \"conversationsId\" does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.bulkUpdate (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:882:12)\n    at Function.update (/server/node_modules/sequelize/lib/model.js:3175:28)"}}
{"message":"column \"conversationsId\" does not exist","time":1695836870500,"stackTrace":{"error":"SequelizeDatabaseError: column \"conversationsId\" does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.bulkUpdate (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:882:12)\n    at Function.update (/server/node_modules/sequelize/lib/model.js:3175:28)"}}
{"message":"column \"conversationsId\" does not exist","time":1695836974665,"stackTrace":{"error":"SequelizeDatabaseError: column \"conversationsId\" does not exist\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:386:16)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.bulkUpdate (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:882:12)\n    at Function.update (/server/node_modules/sequelize/lib/model.js:3175:28)"}}
{"message":"UsersInConversations is not associated to Users!","time":1695837145131,"stackTrace":{"error":"SequelizeEagerLoadingError: UsersInConversations is not associated to Users!\n    at Function._getIncludedAssociation (/server/node_modules/sequelize/lib/model.js:710:13)\n    at Function._validateIncludedElement (/server/node_modules/sequelize/lib/model.js:614:53)\n    at /server/node_modules/sequelize/lib/model.js:509:37\n    at Array.map (<anonymous>)\n    at Function._validateIncludedElements (/server/node_modules/sequelize/lib/model.js:504:39)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1723:12)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"chat is not defined","time":1695837229610,"stackTrace":{"error":"ReferenceError: chat is not defined\n    at _callee5$ (/server/src/controllers/chatController.js:194:74)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/chatController.js:17:103)\n    at _next (/server/src/controllers/chatController.js:19:194)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"chat is not defined","time":1695837305934,"stackTrace":{"error":"ReferenceError: chat is not defined\n    at _callee5$ (/server/src/controllers/chatController.js:191:74)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/chatController.js:17:103)\n    at _next (/server/src/controllers/chatController.js:19:194)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"chat is not defined","time":1695837456809,"stackTrace":{"error":"ReferenceError: chat is not defined\n    at _callee5$ (/server/src/controllers/chatController.js:191:74)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/chatController.js:17:103)\n    at _next (/server/src/controllers/chatController.js:19:194)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"chat is not defined","time":1695837527832,"stackTrace":{"error":"ReferenceError: chat is not defined\n    at _callee5$ (/server/src/controllers/chatController.js:191:74)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/chatController.js:17:103)\n    at _next (/server/src/controllers/chatController.js:19:194)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"chat is not defined","time":1695838280975,"stackTrace":{"error":"ReferenceError: chat is not defined\n    at _callee5$ (/server/src/controllers/chatController.js:190:82)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/controllers/chatController.js:17:103)\n    at _next (/server/src/controllers/chatController.js:19:194)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"}}
{"message":"token error","time":1695840013008,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695840013030,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695840121171,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695840121176,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695840123961,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695840123967,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911613444,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911613482,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911781640,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911781649,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911783129,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911783131,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911783134,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911783152,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911808726,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911808749,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911808755,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911808761,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911946239,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911952174,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1695911952182,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696000970984,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696000973681,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1696001022990,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1696001022993,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1696001025788,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1696001025791,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1696001025796,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696001054051,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696001065196,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696001407814,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696001956274,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696002214294,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696002216963,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696002229009,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696002245941,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"need token","time":1696002782621,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"need token","time":1696002782633,"code":408,"stackTrace":{"error":"TokenError: need token\n    at _callee$ (/server/src/middlewares/checkToken.js:9:17)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696003629427,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696003682219,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696003734484,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696003873278,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"WHERE parameter \"id\" has invalid \"undefined\" value","time":1696004522637,"stackTrace":{"error":"Error: WHERE parameter \"id\" has invalid \"undefined\" value\n    at PostgresQueryGenerator.whereItemQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2269:13)\n    at /server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2258:25\n    at Array.forEach (<anonymous>)\n    at PostgresQueryGenerator.whereItemsQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2256:35)\n    at PostgresQueryGenerator.getWhereConditions (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:2674:19)\n    at PostgresQueryGenerator.selectQuery (/server/node_modules/sequelize/lib/dialects/abstract/query-generator.js:1325:28)\n    at PostgresQueryInterface.select (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:954:27)\n    at Function.findAll (/server/node_modules/sequelize/lib/model.js:1753:47)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at Function.findOne (/server/node_modules/sequelize/lib/model.js:1917:12)"}}
{"message":"token error","time":1696004594503,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee2$ (/server/src/middlewares/checkToken.js:38:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:29:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/server/node_modules/express/lib/router/index.js:317:13)\n    at /server/node_modules/express/lib/router/index.js:284:7\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)\n    at next (/server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/server/node_modules/express/lib/router/index.js:174:3)"}}
{"message":"token error","time":1696004594515,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee2$ (/server/src/middlewares/checkToken.js:38:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:29:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/server/node_modules/express/lib/router/index.js:317:13)\n    at /server/node_modules/express/lib/router/index.js:284:7\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)\n    at next (/server/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/server/node_modules/express/lib/router/index.js:174:3)"}}
{"message":"token error","time":1696004597427,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696004597436,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696005785315,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696005785327,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696005792098,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696005792102,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696008207713,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696008207723,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696008210514,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696008210518,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"Validation error","time":1696008468384,"stackTrace":{"error":"SequelizeUniqueConstraintError: Validation error\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:344:18)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.insert (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:749:21)\n    at Messages.save (/server/node_modules/sequelize/lib/model.js:3954:35)\n    at Function.create (/server/node_modules/sequelize/lib/model.js:2207:12)"}}
{"message":"Validation error","time":1696008480734,"stackTrace":{"error":"SequelizeUniqueConstraintError: Validation error\n    at Query.formatError (/server/node_modules/sequelize/lib/dialects/postgres/query.js:344:18)\n    at Query.run (/server/node_modules/sequelize/lib/dialects/postgres/query.js:87:18)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at /server/node_modules/sequelize/lib/sequelize.js:619:16\n    at PostgresQueryInterface.insert (/server/node_modules/sequelize/lib/dialects/abstract/query-interface.js:749:21)\n    at Messages.save (/server/node_modules/sequelize/lib/model.js:3954:35)\n    at Function.create (/server/node_modules/sequelize/lib/model.js:2207:12)"}}
{"message":"token error","time":1696263954538,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696263957014,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696263957019,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696263957021,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696263957028,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696267761083,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696267761090,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696267762423,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696267762425,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
{"message":"token error","time":1696267762435,"code":408,"stackTrace":{"error":"TokenError: token error\n    at _callee$ (/server/src/middlewares/checkToken.js:25:10)\n    at tryCatch (/server/node_modules/regenerator-runtime/runtime.js:63:40)\n    at Generator.invoke [as _invoke] (/server/node_modules/regenerator-runtime/runtime.js:293:22)\n    at Generator.next (/server/node_modules/regenerator-runtime/runtime.js:118:21)\n    at asyncGeneratorStep (/server/src/middlewares/checkToken.js:3:103)\n    at _next (/server/src/middlewares/checkToken.js:5:194)\n    at /server/src/middlewares/checkToken.js:5:364\n    at new Promise (<anonymous>)\n    at /server/src/middlewares/checkToken.js:5:97\n    at /server/src/middlewares/checkToken.js:6:1\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at next (/server/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/server/node_modules/express/lib/router/route.js:112:3)\n    at Layer.handle [as handle_request] (/server/node_modules/express/lib/router/layer.js:95:5)\n    at /server/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/server/node_modules/express/lib/router/index.js:335:12)"}}
